scripts/split_storage_migration.sh

```diff
@@ -54,32 +99,31 @@ function run_with_trie_changes {
 function init_cold_storage {
   # Switch to migration mode
   echo 'Starting initial migration run'
+  echo 'Expect the migration to take a long time (>1 hour).'
+  echo 'Do not interrupt. Any interruption will undo the migration process.'
   cp $NEAR_HOME/config.json.migration $NEAR_HOME/config.json
-  sudo systemctl restart neard
-
-  # Wait for migration success status
-  while true
-  do
-    head=$(curl 0.0.0.0:3030/metrics | grep 'block_height_head' | tail -n1 | awk '{print $2}')
-    cold_head=$(curl 0.0.0.0:3030/metrics | grep 'cold_head_height' | tail -n1 | awk '{print $2}')
-
-    echo "$head"
-    echo "$cold_head"
-
-    if [[ -n "$head" && -n "$cold_head" ]]
-    then
-      if ((head - cold_head < 1000))
-      then
-        break
-      fi
-    fi
 
+  echo "Restarting the systemd service '$service_name'"
+  sudo systemctl restart $service_name
+
+  # Wait for the migration to complete
+  while [[ -z "$head" || -z "$cold_head" || $(($head - $cold_head)) -ge 1000 ]]; do
+    # Get data from the metrics page of a localhost node that is assumed to listen on port 3030.
+  metrics_url="http://0.0.0.0:3030/metrics"
+    head=$(curl "$metrics_url" --silent | grep 'block_height_head' | tail -n1 | awk '{print $2}')
+    cold_head=$(curl "$metrics_url" --silent | grep 'cold_head_height' | tail -n1 | awk '{print $2}')
+
+    echo "Head of hot storage: '$head'"
+    echo "Head of cold storage: '$cold_head'"
```

Comment:
Maybe add notice about not interrupting the node here also. If cold head is not set print warning that cold storage migration isn't finished and restarting the node will undo the progress.